packages/icons/src/companies/index.js

```diff
@@ -20,7 +20,9 @@ const companyLookup = {
 };
 
 function getCompanyIcon(name) {
-  const icon = companyLookup[name.toLowerCase()];
+  const icon =
+    companyLookup[name.toLowerCase()] ||
+    companyLookup[name.toLowerCase().slice(4)];
```

Comment:
Add the description of the PR as a comment here, so that we'll remember where this logic came from.